index.html

```diff
@@ -7,14 +7,150 @@
 
+        <nav class="mobile-navigation">
+            <div class="navbar-tops">
+                <button class="appointment-button"> Book an appointment</button>
+                <button class="menu-button" onclick="openMenu()">
+                    &#9776;
+                  </button>
+            </div>
+
+            <h1 class="phd-text text-center col-12 col-md-6">Maria Hope, Ph.D. - Psychologist</h1>
+            <div class="navbar-nav mobile-menu mobile-menu-hidden" id="mobile-menu">
+                <a href="#">Home</a>
+                <a href="#">About</a>
+                <a href="#">Services</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+              </div>
+        </nav>
+
+        <!-- Desktop Nav Bar-->
+
+        <nav class="desktop-navigation">
+            <div class="maria-phd">
+                 <h1 class="phd-text ">Maria Hope, Ph.D. - Psychologist</h1>
+            </div>
+
+            <div class="nav-bar-right">
+                <button class="appointment-button"> Book an appointment</button>
+                
+                <div class="bar-extended">
+                    <a href="#">Home</a>
+                    <a href="#">About</a>
+                    <a href="#">Services</a>
+                    <a href="#">Blog</a>
+                    <a href="#">Contact</a>
+                  </div>
+            </div>
+        </nav>
+    </header>
+
+<!-- Main content -->
+
+    <main class="primary-content">
+        <section class="hero-main">
+            <img class="col-12" src="./img/suculent.webp" alt="Succulent plant">
+
+            <section class="mint-container col-10 offset-1 col-md-8 offset-md-2">
+                <h2>Online Therapy for Emotional Health</h2>
+                <p>I am a description. Click here to edit.</p> 
+            </section>
+        </section>
+
+        <section class="about-maria">
+            <article class="about-psychologist">
+                <div class="col-md-8 offset-md-2" >
+                    <h3 class="col-12 text-center">About</h3>
+                    <h2 class="col-12 text-center">Maria Hope</h2>
+                    <p class="col-10 offset-1 col-md-8 offset-md-2"> I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to add your own content and make changes to the font. Feel free to drag and drop me anywhere you like on your page. I’m a great place for you to tell a story and let your users know a little more about you.</p> 
+                </div>
+            </article>
+
+ <!-- Articles boxes sections -->
+ </article>
+        
+        <section>
+            <img class="col-10 maria-maria align-self-center" src="./img/psychologist_woman.webp" alt="Maria Hope Psychologist">
+        </section>
+
+        <section class="services-section"> 
+            <article class="text-w-box box-n-1 col-md-4 ">
+                <div class=" offset-1 col-md-8 offset-md-2">
```

Comment:
Again, don't use `col-...` classes on both these elements. Only on the one directly after `row`.